conceptnet5/vectors/query.py

```diff
@@ -187,6 +196,12 @@ def expanded_vector(self, terms, limit_per_term=10, include_neighbors=True):
             self.expand_terms(terms, limit_per_term, include_neighbors)
         )
 
+    def get_fasttext_vector(self, query):
+        """Look up a vector for a query in the fastText model."""
+        label = query.split('/')[-1]
```

Comment:
I think it makes sense to apply `uri_prefix` to URIs being looked up.

We shouldn't have to apply it at the point where we're loading the DataFrame, as I don't see a plausible way that four- or five-part URIs would show up in the data there. If we get extra slashes in a URI, it came from somewhere else -- possibly extra slashes in the input data. For example, fastText terms can contain slashes.

The solution to that would be to standardize URIs appropriately, where slashes should turn into underscores, instead of just concatenating strings so slashes pass through. (Line 92 of query.py is currently doing this the naive concatenating way, and I know that's my fault).